src/components/central-program/program-data-overview-table.js

```diff
@@ -3,28 +3,46 @@ import PropTypes from "prop-types"
 import BootstrapTable from "react-bootstrap-table-next"
 import ToolkitProvider from "react-bootstrap-table2-toolkit"
 import { formatToUSD, formatFTE, deltaPrefix } from "../table-utilities"
+import { getColumnsByDataField } from "../../utilities/content-utilities"
+
+const DESCRIPTION = "description"
+const VALUE = "value"
+const STAFF = "staff"
+
+const CHANGE = "change_from_previous_year"
+const SPENDING = "spending"
+const BUDGET = "budget"
+const BALANCE = "budget_balance"
+const STAFF_FTE = "eoy_total_fte"
+const STAFF_POSITIONS = "eoy_total_positions"
```

Comment:
Not sure if these constants make sense. Was just trying to put some distance from the names in Contentful and all the places where we reference them below.

Reply:
seems reasonable to me!